packages/components/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## Unreleased
 
 ### Bug Fixes
-
+-   `Modal` : Modal dialog small improvement for elementShouldBeHidden - PR ([#65941](https://github.com/WordPress/gutenberg/pull/65941)).
```

Comment:
The changelog entry should follow a specific format:
- Please remove the space before the colon.
- Please remove the `- PR `.
- I'd consider to move the entry to the `Enhancements` section as it's not a bug fix.